docs/pages/versions/unversioned/react-native/flatlist.md

```diff
@@ -178,19 +165,20 @@ Provides additional metadata like `index` if you need it, as well as a more gene
 | -------- | -------- |
 | function | Yes      |
 
-- `item` (Object): The item from `data` being rendered.
-- `index` (number): The index corresponding to this item in the `data` array.
-- `separators` (Object)
+  - `item` (Object): The item from `data` being rendered.
+  - `index` (number): The index corresponding to this item in the `data` array.
+  - `separators` (Object)
```

Comment:
I would change L170-L177 to
```
  - `separators` (Object)
    - `highlight` (Function)
    - `unhighlight` (Function)
    - `updateProps` (Function)
      - `select` (enum('leading', 'trailing'))
      - `newProps` (Object)
```

Reply:
Ah so as it is written and how you have it creates an error in one of the markdown validation rules.  

I'm wasn't sure what the nested list rules were for the docs site parser so I made a new block.

![Screenshot from 2019-10-04 00-33-03](https://user-images.githubusercontent.com/7072452/66181692-b8d7f680-e63f-11e9-9356-0c192602c742.png)

Reply:
It looks like the 3rd indented block needed additional spacing (4 if the prev section was 2, or 6 if the prev block was 4 spaces).    I'll  go ahead and fix that so you can close the issue. 

```
- `item` (Object): The item from `data` being rendered.  
- `index` (number): The index corresponding to this item in the `data` array.  
- `separators` (Object)  
  - `highlight` (Function)  
  - `unhighlight` (Function)  
  - `updateProps` (Function)  
        - `select` (enum('leading', 'trailing'))  
        - `newProps` (Object)  
```

It might be a worthwhile task to setup a precommit or precompile markdown linting tasks since that type thing is pretty hard to catch visually.  I only saw it because I was sitting in the Github editor but normally that wouldn't be the case.